craco.config.cjs

```diff
@@ -93,6 +94,10 @@ module.exports = {
       // See https://vanilla-extract.style/documentation/integrations/webpack/#identifiers for docs.
       // See https://github.com/vanilla-extract-css/vanilla-extract/issues/771#issuecomment-1249524366.
       new VanillaExtractPlugin({ identifiers: 'short' }),
+      new RetryChunkLoadPlugin({
+        retryDelay: 2000,
```

Comment:
This should be a function to add exponential backoff.

Reply:
Not possible in their repo, so I'll have to fork if we want that.